src/mongocxx/collection.cpp

```diff
@@ -83,6 +83,12 @@ collection::collection(const database& database, bsoncxx::stdx::string_view coll
           database.name(), database._impl->client_impl, collection_name.data())) {
 }
 
+collection::collection(const database& database, bsoncxx::stdx::string_view collection_name,
```

Comment:
Would it be worth switching `mongocxx::stdx` instead of `bsoncxx::stdx` throughout?
I have a pr open to change this everywhere, but as long as you're touching these files, it might be worth it.